index.html

```diff
@@ -669,6 +669,30 @@ <h2>A project for first-time contributions</h2>
           </div>
         </div>
         <!-- ========= Card 12 END =========  -->
+        <div class="card">
+          <p class="name">Vera</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com/your_user_handle" target="_blank">tschanzvera</a>
```

Comment:
as previously mentioned for your github handle, please also update this link with your actual twitter handle